cli/tests/conftest.py

```diff
@@ -158,7 +163,9 @@ def example_messages() -> list[Message]:
 @pytest.fixture()
 def example_messages_df(example_messages):
     """Test input data in a DataFrame."""
-    return pd.DataFrame.from_records([vars(im) for im in example_messages])
+    messages_df = pd.DataFrame.from_records([vars(im) for im in example_messages])
+    messages_df["pseudo_patient_id"] = 0
```

Comment:
I'd expect the default case for this to not have a `pseudo_patient_id` so probably worth setting to `None` unless we're testing the use of `pseudo_patient_id` specifically

Reply:
[d920af6](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/d920af6f3c423992677f33c5f774a7b01a7e24c8)